lib/push.js

```diff
@@ -218,7 +218,7 @@ module.exports = function (log, db, config) {
         }
       }))
       var options = { data: data, TTL: TTL_DEVICE_DISCONNECTED }
-      return this.pushToDevice(uid, idToDisconnect, 'deviceDisconnected', options)
```

Comment:
Reading #1777, it sounds (to me) that the `idToDisconnect` is still needed, since all devices use that to check if they are being disconnected.